physics/cires_ugwp_initialize_v1.F90

```diff
@@ -0,0 +1,805 @@
+!===============================
+! cu-cires ugwp-scheme
+!    initialization of selected 
+!  init gw-solvers (1,2,3,4)
+!  init gw-source specifications
+!  init gw-background dissipation
+!==============================
+!
+! Part-0   specifications of common constants, limiters and "criiical" values
+! 
+!
+    
+    module ugwp_common_v1
+!
+!     use machine,  only : kind_phys
+!     use physcons, only : pi => con_pi, grav => con_g, rd => con_rd,   &
+!                          rv => con_rv, cpd => con_cp, fv => con_fvirt,&
+!                          arad => con_rerth
+     implicit none
+
+      real, parameter ::  grav =9.81, cpd = 1004.
+      real, parameter ::  rd = 287.0 , rv =461.5      
+      real, parameter ::  grav2 = grav + grav
+      real, parameter ::  rgrav = 1.0/grav, rgrav2= rgrav*rgrav
+           
+      real, parameter ::  fv   = rv/rd - 1.0  
+      real, parameter ::  rdi  = 1.0 / rd, rcpd = 1./cpd, rcpd2 = 0.5/cpd  
+      real, parameter ::  gor  = grav/rd
+      real, parameter ::  gr2  = grav*gor
+      real, parameter ::  grcp = grav*rcpd, gocp = grcp
+      real, parameter ::  rcpdl    = cpd*rgrav                         ! 1/[g/cp]  == cp/g
+      real, parameter ::  grav2cpd = grav*grcp                         !  g*(g/cp)= g^2/cp
```

Comment:
Even some of these derived constants have CCPP standard names, BTW. Where they don't, and they must be calculated locally, they will of course need to no longer be defined as parameters.

Reply:
I had found that the derived ratio "fv" had the CCPP standard name "con_fvirt", and I used it in my previous revision from last week, where I did recalculate them locally (not as parameters).  For now I would like to leave ugwp_common_v1 as-is even though I'm not using any physical constants from it.  We can get rid of unused constants for our final ugwp_v1 version.  I hope that's okay.

I just found some more ratios in ccpp/physics/physics/physcons.F90 and I could pass them through GFS_typedefs.meta later if necessary.  Is physcons.F90 correct source?  Also, are we free to add our own ratios to physcons.F90?  (BTW, why aren't more of the ratios already in GFS_typedefs.meta?)